indexers/mainnet/consensus/src/mappings/helper.ts

```diff
@@ -27,3 +30,17 @@ export const getBlockAuthor = (block: SubstrateBlock): string => {
   }
   return DEFAULT_ACCOUNT_ID;
 };
+
+export const parseDataToCid = (data: string): ParsedArgs => {
+  let cid: Cid = undefined;
+  let modifiedArgs: ModifiedArgs = undefined;
+  try {
+    const node = decodeNode(hexToUint8Array(data));
```

Comment:
This parse works for the IPLD nodes sent as hex string but this is now deprecated in favor of raw binary data. This [PR](https://github.com/autonomys/auto-sdk/pull/171) implemented the change.

Reply:
Indeed data shouldn't be considered a `string` always should be able to be `Bytes` too

Reply:
Do you have an extrinsic that has the data as per the new format for me to test?